custom_components/easee/sensor.py

```diff
@@ -53,11 +60,18 @@ async def async_setup_entry(hass, entry, async_add_entities):
                 )
 
         monitored_days = config.options.get(MEASURED_CONSUMPTION_DAYS, [])
+        consumption_unit = "kWh"
```

Comment:
These 3 lines can be written as one line

Reply:
like

`consumption_unit = CUSTOM_UNITS_TABLE[consumption_unit] if consumption_unit in custom_units else "kWh"`

or something smarter?

Reply:
I think so.